README.md

```diff
@@ -15,14 +15,22 @@ Have fun 😃
 `/gen-build`: Generates up to 12 builds for your selected game version. *Pro Tip: You can also exclude inline-bikes if you don't like them 🙃*
 
 ## Building the bot by yourself
-If you want for whatever reason want to build the bot by yourself you can do so by pulling this repository and configure it with your bot token.
-Create a config.json inside the root folder of the project and fill it with the following.
+If, for whatever reason you want to build the bot by yourself, you can do so by pulling this repository and configuring it with your bot token. Your bot token can be found in the [Discord Developer Portal](https://discord.com/developers/applications) under your bot's name, select the "Bot" option under settings and then you should be able to see it underneath the bots username.
+
+If you want to build a brand new bot, you'll need to give the bot the "Bot" and "Applications.Commands" permissions under scope.
+Under "Bot Permissions", you'll need to enable "Read Messages/View Channels", "Send Messages", and "Embed Links".
```

Comment:
This should be redundant. Correct me if I'm wrong. The Scope that you select should automatically pre-select the permissions.

Reply:
No, none of them are pre-selected. I'm gonna edit this to make it more clear that this is for the link for the bot to join.